.helm/ecamp3/templates/browserless_deployment.yaml

```diff
@@ -49,5 +49,19 @@ spec:
             periodSeconds: 10
             timeoutSeconds: 5
           resources:
-            {{- toYaml .Values.resources | nindent 12 }}
+            {{- toYaml .Values.browserless.resources | nindent 12 }}
+          envFrom:
+            - configMapRef:
+                name: {{ include "browserless.name" . }}-configmap
+          env:
+            - name: PREBOOT_CHROME
+              value: "true"
+            - name: KEEP_ALIVE
+              value: "true"
+            - name: ENABLE_DEBUGGER
+              value: "false"
+            - name: FUNCTION_ENABLE_INCOGNITO_MODE
+              value: "true"
+            - name: DISABLED_FEATURES
+              value: '["downloadEndpoint","functionEndpoint","pdfEndpoint","screencastEndpoint","scrapeEndpoint","statsEndpoint","workspaces"]'
```

Comment:
If we don't use pdfEndpoint, what do we use?

Reply:
The pdfEndpoint is the [/pdf REST](https://docs.browserless.io/docs/pdf.html) endpoint. We're not using the REST endpoint but direct [WebSocket connection](https://docs.browserless.io/docs/puppeteer-library.html) from puppeteer.

Reply:
The pdfEndpoint is the [/pdf REST](https://docs.browserless.io/docs/pdf.html) endpoint. We're not using the REST endpoint but direct [WebSocket connection](https://docs.browserless.io/docs/puppeteer-library.html) from puppeteer.